axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/Checksum.java

```diff
@@ -13,14 +13,29 @@
 import java.util.zip.CRC32;
 
 /**
+ * Calculates the CRC32 checksum of a given {@link ByteBuffer}, without loading the full content in memory.
+ *
  * @author Marc Gathier
+ * @author Sara Pellegrini
+ * @author Milan Savic
  */
 public class Checksum {
+
+    private static final int oneMB = 1024 * 1024;
     private final CRC32 crc32;
+    private final int batch;
 
+    /**
+     * Constructs an instance that updates the checksum value loading data in chucks of 4 MBs.
+     */
     public Checksum() {
+        this(4 * oneMB);
```

Comment:
This can be the static final int, no need to compute it. (`fourMB`?) `oneMB` is used nowhere else.